samples/Vue/basic.vue

```diff
@@ -0,0 +1,21 @@
+<style>
+.red {
+  color: #f00;
+}
+</style>
+
+<template>
+<div>
+  <h2 v-class="red">{{msg}}</h2>
+</div>
+</template>
+
+<script>
+module.exports = {
```

Comment:
Do Vue files always contain a JavaScript component? (I'm trying to see if we should tag this as `programming` or `markup`.)